<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataModelingGUI</class>
 <widget class="QMainWindow" name="DataModelingGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>1386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataModelingGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="dataComboBox"/>
        </item>
        <item>
         <widget class="QTableWidget" name="dataTableWidget">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plotTab">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="plotControlsLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="plotXLabel">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="plotXListWidget">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <item>
             <property name="text">
              <string>a</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>b</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>c</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>d</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>e</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="plotYLabel">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="plotYListWidget">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="plotTypeComboBox">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>1D Plot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Scatter</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PairPlot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BoxPlot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ViolinPlot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Autocorrelation</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Partial Autocorrelation</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="plotButton">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="plotPlotLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="functionTab">
       <attribute name="title">
        <string>Function</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="functionTypeComboBox">
            <item>
             <property name="text">
              <string>--Function Type--</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Manual</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="functionVarComboBox"/>
          </item>
          <item>
           <widget class="QLineEdit" name="functionNameLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="functionComputeButton">
            <property name="text">
             <string>Compute</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="functionCustomLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Enter custom function here...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="olsTab">
       <attribute name="title">
        <string>OLS</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="mlTab">
       <attribute name="title">
        <string>ML</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>298</width>
          <height>378</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="mlTypeComboBox">
           <item>
            <property name="text">
             <string>Classification</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Regression</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mlMethodComboBox">
           <item>
            <property name="text">
             <string>Ridge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Logistic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SVM</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mlTargetComboBox"/>
         </item>
         <item>
          <widget class="QListWidget" name="mlSourcesListWidget"/>
         </item>
         <item>
          <widget class="QComboBox" name="mlCVComboBox">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LOO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KFold-Naive</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KFold-Stratified</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KFold-Temporal</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Crossvalidation Param</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="mlRegParamLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Regularization Param</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="mlKFoldParamLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Estimate Regularization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Fit Model</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_CSV"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_CSV">
   <property name="text">
    <string>Load CSV...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
